### ISSUE TITLE
[Feature request] In Editor|"Search for" (Ctrl+F) to display the currently highlighted position and the total found occurrences count

### ISSUE THREAD

**Issue by darkred** [Tampermonkey/tampermonkey#587]
Using TM 4.7 stable/4.8.5835 beta in Chrome 68 and Firefox 61, 

It would be great if in Editor|"Search for" (Ctrl+F)
it would display the currently highlighted position and the total found occurrences count.

E.g. let's say that you search for the string inside a script,
and the currently highlighted position is 2 and the script contains the string 3 times in total:
in such case it would display: `[ 2 of 3 ]`.

Thank you

**Comment by dnknn** [comment]
@darkred @derjanb 

> it would display the currently highlighted position (Current number)

👎 This is completely unnecessary and meaningless. The important thing is that it will affect the performance of the editor (or the performance or speed of the search)

And you can view the current position with the scroll-bar, 

<details>
<summary>And you can also customize the CSS style</summary>

![212_22204023](https://user-images.githubusercontent.com/10995302/108709826-31e62f80-754e-11eb-8829-6ef04f09bff9.png)

```css
.CodeMirror-search-match{width:47px !important;border-left:1px solid #ffa50070}

.CodeMirror-search-field{color:blue!important;font-size:20px;
max-width:31vw!important;border:1px #bbb solid !important}
.CodeMirror-search-field:focus{outline:auto #4d90fe 5px;background:white}
.CodeMirror-search-field~button:focus{outline:auto #4d90fe 5px}
.CodeMirror-dialog-bottom .CodeMirror-search-close-button{padding:1px 45px;margin-left:13px}


.CodeMirror-vscrollbar:focus{outline:auto #4d90fe 5px;background:#dfc9c9}
.CodeMirror-vscrollbar:focus::-webkit-scrollbar{background:#f1eded}
.CodeMirror-vscrollbar:focus::-webkit-scrollbar-button{background:#aaa}
.CodeMirror-vscrollbar:focus::-webkit-scrollbar-thumb{background:#7936a6}
.CodeMirror-vscrollbar:focus::-webkit-scrollbar-thumb:hover{background:#4d097c}

.CodeMirror-vscrollbar::-webkit-scrollbar{width:33px;background:#ebd8d8}
.CodeMirror-vscrollbar::-webkit-scrollbar-thumb{background:#6d857c}
.CodeMirror-vscrollbar::-webkit-scrollbar-button{background:#ccc}
.CodeMirror-vscrollbar::-webkit-scrollbar-thumb:hover{background:#b2720f}
.CodeMirror-vscrollbar::-webkit-scrollbar-thumb:active{background:#8d4bbb}

```


</details>

---

> the total found occurrences count.

💪👍 But this must have it, this is the basic function of search, and users can directly see the total number of searches.

The screenshot below is from Stylus' CodeMirror editor

![212_22204407](https://user-images.githubusercontent.com/10995302/108710205-b6d14900-754e-11eb-9760-bfd629dab278.png)

**Comment by darkred** [comment]
> This is completely unnecessary and meaningless. 

Keep your opinion to yourself. 
With https://github.com/Tampermonkey/tampermonkey/issues/1151#issuecomment-783327486 I was referring to @derjanb , he is the one to decide, what suggestion is useful and what's not.

**Comment by dnknn** [comment]
> Keep your opinion to yourself.

Well, there is no hostile behavior, please don't take offense, sorry... ^_^

**Comment by dnknn** [comment]
Hi @derjanb 

Two additional suggestions about the `Ctrl + F` search bar: 

## 1. If asy to achieve, please support instant search.

- it can reduce pressing `Enter` or click the OK button.

## 2. Able to remember the last search term/Keyword.

- This is very convenient for simultaneous searches between multiple scripts.

----

The above 2 functions can easily realize `Ctrl+F` to directly follow the last search, 
In other words, you no longer need `Ctrl+V` and `Enter` to confirm for persistent search, 


> For details, refer to the effect of the search bar in the [Stylus](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) editor
